internal/manifest/manifest.go

```diff
@@ -65,33 +70,37 @@ func (p Policy) Hash() HexString {
 
 // Validate checks the validity of all fields in the reference values.
 func (r ReferenceValues) Validate() error {
-	if r.AKS != nil {
-		if err := r.AKS.Validate(); err != nil {
-			return fmt.Errorf("validating AKS reference values: %w", err)
+	if r.SNP != nil {
+		for _, v := range r.SNP {
+			if err := v.Validate(); err != nil {
+				return fmt.Errorf("validating SNP reference values: %w", err)
+			}
```

Comment:
```suggestion
	for _, v := range r.SNP {
		if err := v.Validate(); err != nil {
			return fmt.Errorf("validating SNP reference values: %w", err)
```

The same change can be done for TDX.